scss/utilities/_variables.scss

```diff
@@ -38,3 +35,11 @@ $transition: 0.14s ease-in-out;
         opacity: 0.8;
     }
 }
+
+
+/* Consultmate variables */
```

Comment:
We should stay away from standardizing these viewports, they are too design/project dependent.